modules/ibi-create-standalone-config-iso.adoc

```diff
@@ -0,0 +1,227 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-edge-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-standalone-config-iso_{context}"]
+= Deploying a standalone {sno} cluster using the {product-title} installation program
+
+For standalone clusters, you can use the {product-title} installation program to configure and deploy a host that you preinstalled with an image-based installation. To configure the target host with site-specific details, you must create the following resources:
+
+* The `install-config.yaml` installation manifest.
+* The `ImageBasedConfig` custom resource.
```

Comment:
I think we should not use the term `ImageBasedConfig custom resource`, I would just call it just `imagebased-config.yaml manifest`, because although it has a `Kind` it's just there to define this config's specification. It is never applied to a cluster in any way.

Reply:
Updated to `image-based-config.yaml` manifest or `image-based-installation-config.yaml` manifest in the docs.